packages/editor/src/components/post-format/index.js

```diff
@@ -57,20 +57,17 @@ export default function PostFormat() {
 			return {
 				postFormat: _postFormat ?? 'standard',
 				suggestedFormat: getSuggestedPostFormat(),
-				// Ensure current format is always in the set.
-				// The current format may not be a format supported by the theme.
-				supportedFormats: union(
-					[ _postFormat ],
-					get( themeSupports, [ 'formats' ], [] )
-				),
+				supportedFormats: themeSupports.formats,
 			};
 		},
 		[]
 	);
 
-	const formats = POST_FORMATS.filter( ( format ) =>
-		includes( supportedFormats, format.id )
-	);
+	const formats = POST_FORMATS.filter( ( format ) => {
```

Comment:
There should be a comment here explaining that the current `postFormat` may not always be present in `supportedFormats`, similar to the removed comment above.

Reply:
Added the comment.